app/controllers/automation_manager_controller.rb

```diff
@@ -439,10 +439,6 @@ def empty_managed_group_record?(inventory_group_record)
     inventory_group_record.try(:id).nil?
   end
 
-  def valid_managed_group_record?(inventory_group_record)
-    inventory_group_record.try(:id)
```

Comment:
I can see this method used:
```
devil - [~/Projects/manageiq-ui-classic] (master)$ grep -r valid_managed_group_record app/
app/controllers/automation_manager_controller.rb:    elsif valid_managed_group_record?(@inventory_group_record)
app/controllers/automation_manager_controller.rb:  def valid_managed_group_record?(inventory_group_record)
app/controllers/automation_manager_controller.rb:    if valid_managed_group_record?(@inventory_group_record)
```

Reply:
@kraldavi Looks like you removed the wrong method .. the commit message says  `empty_managed_group_record?` (`empty` vs `valid`), and `empty_managed_group_record?` is indeed dead.